lab11_3/src/main.js

```diff
@@ -0,0 +1,47 @@
+// main.js
+const root = document.querySelector('main');
+ReactDOM.render(<App />, root);
+
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { useState } from 'react';
+
+import './style.css';
+
+const App = () => {
+  const [number, setNumber] = useState(getRandom(1, 1000));
+  const [guessedNumber, setGuessedNumber] = useState();
+  const [message, setMessage] = useState('');
+  return (
+    <div id="app-container">
+      <strong>{message}</strong><br></br>
+      <label>The number is????</label><br></br>
+      <input value={guessedNumber} onChange={e => onChange(e.target.value)} />
+     
+    </div>
+  );
+
+  function onChange(guess) {
+    if (guess == number) {
+      setMessage('YOU GOT IT');
+      return;

```

Comment:
Why do you return without reaching `setGuessedNumber`?

Reply:
correct it was a left over